tron/core/recovery.py

```diff
@@ -19,13 +20,17 @@ def filter_action_runs_needing_recovery(action_runs):
 
 def filter_recoverable_action_runs(action_runs):
     """
-    Given a list of action_runs, create a filtered list that only includes those that can be recovered
-    For now, the only test is whether the run is an SSHActionRun
+    Given a list of action_runs, return those that can be recovered.
+    SSHActionRuns and MesosActionRuns are returned separately.
```

Comment:
if you're going to change what this function does, can you rename it? this doesn't really do any filtering anymore. just ``group_by_actionrun_type`` or something?

Reply:
Sure, done.